sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -99,10 +99,11 @@ def test_classification_toy(loss, global_random_seed):
 def test_classification_synthetic(loss, global_random_seed):
     # Test GradientBoostingClassifier on synthetic dataset used by
     # Hastie et al. in ESLII - Figure 10.9
-    X, y = datasets.make_hastie_10_2(n_samples=12000, random_state=global_random_seed)
+    X, y = datasets.make_hastie_10_2(n_samples=2000, random_state=global_random_seed)
```

Comment:
We should a comment above. The `12_000` is there because we were running the exact experiment of the ESL book.

Reply:
I second this opinion.

Reply:
already done